android/app/build.gradle

```diff
@@ -170,3 +180,24 @@ gradle.projectsEvaluated {
 if (googleServicesEnabled) {
     apply plugin: 'com.google.gms.google-services'
 }
+
+subprojects { subproject ->
```

Comment:
I think this should go in the sdk build.gradle, so the descending projects are updated when people use the SDK.

Reply:
Dropped.